src/Ledger/ScriptValidation.agda

```diff
@@ -16,7 +16,7 @@ open import Ledger.Crypto
 
 
 module Ledger.ScriptValidation
-  (txs : _) (open TransactionStructure txs)
+  (txs : _) (open TransactionStructure txs hiding (Vote))
```

Comment:
Previously we didn't need to hide `Cert/Rwrd/Mint/Spend`, so why is this now necessary for `Vote`?